ee/keys_ee.go

```diff
@@ -0,0 +1,67 @@
+// +build !oss
+
+/*
+ * Copyright 2021 Dgraph Labs, Inc. All rights reserved.
+ *
+ * Licensed under the Dgraph Community License (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ *     https://github.com/dgraph-io/dgraph/blob/master/licenses/DCL.txt
+ */
+
+package ee
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/dgraph-io/ristretto/z"
+	"github.com/spf13/viper"
+)
+
+// GetKeys returns the ACL and encryption keys as configured by the user
+// through the --acl, --encryption, and --vault flags. On OSS builds,
+// this function always returns an error.
+func GetKeys(config *viper.Viper) (*Keys, error) {
+	keys := &Keys{}
+	var err error
```

Comment:
can this be removed?

Reply:
The first use of `err` assigns directly to a struct:

```go
keys.AclKey, err = ioutil.ReadFile(aclKeyFile)
```

So we can't remove the error declaration, unfortunately.